test/conformance/usm/urUSMDeviceAlloc.cpp

```diff
@@ -2,21 +2,47 @@
 // Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
 // See LICENSE.TXT
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
 #include <uur/fixtures.h>
 
-struct urUSMDeviceAllocTest : uur::urQueueTestWithParam<uur::BoolTestParam> {
+using USMDeviceAllocParams = std::tuple<uur::BoolTestParam, uint32_t, size_t>;
+
+template <typename T>
+inline std::string printUSMDeviceAllocTestString(
```

Comment:
I think this could equally be used for the other USM alloc entry points, maybe it should be in a header as just `USMAllocParams`/`printUSMAllocTestString`?

Reply:
No bother to add these into a common header. Should I also add a `SuccessAlignedAllocations` test for Host and Shared?

Reply:
yeah they should work exactly the same way wrt alignment